src/dialog.tsx

```diff
@@ -20,45 +20,39 @@ import * as styles from './dialog.css';
 import {vars} from './skins/skin-contract.css';
 
 import type {ButtonLink} from './button';
-import type {Theme} from './theme';
 import type {RendersNullableElement} from './utils/types';
 import type {ExclusifyUnion} from './utils/utility-types';
 
-const animationsSupported = (platformOverrides: Theme['platformOverrides']) =>
-    process.env.NODE_ENV !== 'test' && !isRunningAcceptanceTest(platformOverrides);
+const shouldAnimate = () => process.env.NODE_ENV !== 'test' && !isRunningAcceptanceTest();
 
 interface BaseDialogProps {
     className?: string;
     title?: string;
-    icon?: React.ReactElement;
     message: string;
     acceptText?: string;
     onAccept?: () => void;
     destructive?: boolean;
 }
 
-interface AlertProps extends BaseDialogProps {
-    showClose?: boolean;
-}
+export type AlertProps = BaseDialogProps;
 
-interface ConfirmProps extends BaseDialogProps {
-    showClose?: boolean;
-    showCancel?: boolean;
+export interface ConfirmProps extends BaseDialogProps {
     cancelText?: string;
     onCancel?: () => void;
 }
-interface ExtendedDialogProps extends BaseDialogProps {
+
+export interface ExtendedDialogProps extends BaseDialogProps {
+    icon?: React.ReactElement;
     subtitle?: string;
     extra?: React.ReactNode;
-    forceWeb?: boolean;
```

Comment:
Isn't this a breaking change? Should we deprecate them instead?

Reply:
strictly speaking, yes, but there are 0 usages of this in webapp